simplex/simplex_problem.py

```diff
@@ -0,0 +1,28 @@
+"""
+SimplexProblem and relevant functions.
+"""
+
+from dataclasses import dataclass
+from typing import Sequence
+
+import numpy as np
+
+from shared.constraints import SimplexConstraint
+from shared.minimization_problem import MinimizationProblem
+
+
+@dataclass
+class SimplexCallable:
+    """Specific callable to SimplexProblem. Keeps A and b accessible."""
+    A: np.ndarray
```

Comment:
actually, a single constraint only contains one a_i of A
same with b, it is only a single value b_i

We may need to expose methods on the problem to get the entire A and b